test/clj/cider/nrepl/middleware/info_test.clj

```diff
@@ -16,6 +16,28 @@
   [x]
   (:resource (info/file-info x)))
 
+(deftest test-javadoc-url
+  (testing "java 1.6"
```

Comment:
We don't support Java 6 anymore (although Clojure itself does), so this test is not necessary. Plus, it's the same behaviour as in 7.